README.md

```diff
@@ -4,6 +4,7 @@
 - Make sure the one time setup is satisfied. (https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md#one-time-setup)
 - Clone this repo `git clone https://github.com/code-ready/snc.git`
 - `cd <directory_to_cloned_repo>`
+- Change the architecture in the [install-config.yaml](install-config.yaml)
 - `./snc.sh`
 
 ## How to create disk image?
```

Comment:
maybe mention here that disk image creation is not supported for arches other than x86_64 ?

Reply:
Why isn't this step automated? And what's wrong with `create_disk.sh`?

Reply:
I was able to create a disk image tarball on ppc64le. I will automate this with yq and remove the line in README

Reply:
when trying it a while ago - i hit this issue where virt-resize was renaming the partitions. For example on s390x, there are only 2 partitions with id 1 and 4. when copying, virt-resize changed that to 1 and 2 causing issues. In the latest commit I see that logic removed, so it should work now!

Reply:
If create_disk.sh was not working yet, I would not necessarily see this as a blocker for that PR, but this would need to be documented.